### ISSUE TITLE
Add a parameter to filter on entity type.

### ISSUE THREAD

**Issue by raygesualdo** [3Blades/app-backend#152]
Alongside `limit` and `offset`, there should be a parameter (perhaps `entities`) that allows the API consumer to define through which entities to search. Below are a few examples.

This would search only users with the string "foo".
```
/<namespace>/search/?q=foo&entity=users
```

This would search both projects and servers for the keyword "notebook".
```
/<namespace>/search/?q=notebook&entity=projects,servers
```

The return payload for requests filtering on entities would only contain keys for entities being search. For example, the second request above would return the following payload:
```javascript
{
  projects: [
    // all matched projects
  ],
  servers: [
    // all matched servers
  ]
}
```

This is blocked by #151.

**Comment by jkp85** [comment]
@raygesualdo First part of this is already done. There is param called `type` that does exactly that.

**Comment by jkp85** [comment]
@raygesualdo One thing about type parameter, it should be sent like this: `type=servers&type=projects`

**Comment by raygesualdo** [comment]
Hmm, is there anyway to change that to a comma-separated list? Passing that on the front-end is really clunky.

**Comment by jkp85** [comment]
There is, but it requires some work. Above way is default to django.
